client/buildpacks.go

```diff
@@ -161,6 +155,23 @@ func (b *buildpacksFunctionServer) run() (func(), error) {
 	}, nil
 }
 
+func (b *buildpacksFunctionServer) getRuntimeVars() []string {
```

Comment:
This function seems weirdly formatted, can you try running `gofmt` on the the files in this PR? https://stackoverflow.com/questions/13319165/go-fmt-on-a-whole-source-tree

Possibly we should turn on a lint checker as a PR validation...

Reply:
vscode's indentation does not show this weird formatting. Thanks for the pointer. I will fix it